broker/models/test/AppointmentTest.js

```diff
@@ -0,0 +1,34 @@
+const assert = require('chai').assert;
+const app = require('../Appointment');
+const data = require('../../testing/examples');
+
+
+describe('Appointment', function() {
+    it('toFHIR should be equal to toModel', function(){
+      let result = true;
+
+        let obj = JSON.parse(data.Appointment);
+        let input = obj.Appointment;
+        let fhir = app.toFHIR(input);
+
+        let output = app.toModel(fhir);
+        
+        for (let property in input) {
+            //console.log(input.property);
+          if (output.property !== input.property) {
+            console.log(property);
```

Comment:
Is there a more mocha/chai-ic method for displaying information about the test failing?

Reply:
```suggestion
            assert.fail(`Expected ${input.property}; received ${output.property}`);
```
Throwing a failure should break from the test and display why the test failed